featuretools/entityset/entityset.py

```diff
@@ -81,6 +81,100 @@ def __init__(self, id, entities=None, relationships=None, verbose=False):
             child_variable = self[relationship[2]][relationship[3]]
             self.add_relationship(Relationship(parent_variable,
                                                child_variable))
+        self._metadata = None
+
+    @property
+    def metadata(self):
+        '''
```

Comment:
can we add a docstring to the first line of this so it shows up properly in the documentation?